docs/eventing/sources/pingsource.md

```diff
@@ -1,196 +1,261 @@
+This is a tight connection between the producer and consumer.
 
-You can now create the `PingSource` sending an event containing
-`{"message": "Hello world!"}` every minute.
+For each set of ping events that you want to request, create an event source in the same namespace
+as the destination.
 
 {{< tabs name="create-source" default="YAML" >}}
 {{% tab name="YAML" %}}
+Create the event source from stdin by running:
 
 ```shell
-kubectl create -n pingsource-example -f - <<EOF
-apiVersion: sources.knative.dev/v1beta2
+cat <<EOF | kubectl create -f -
+apiVersion: sources.knative.dev/v1alpha2
 kind: PingSource
 metadata:
   name: test-ping-source
 spec:
-  schedule: "*/1 * * * *"
+  schedule: "*/2 * * * *"
   contentType: "application/json"
   data: '{"message": "Hello world!"}'
   sink:
     ref:
-      apiVersion: v1
+      apiVersion: serving.knative.dev/v1
       kind: Service
       name: event-display
 EOF
 ```
-
 {{< /tab >}}
 
 {{% tab name="kn" %}}
-Notice that the namespace is specified in two places in the command in `--namespace` and the `--sink` hostname
+Create the event source from the `ping-source.yaml` file by running:
+
 ```shell
 kn source ping create test-ping-source \
-  --namespace pingsource-example \
-  --schedule "*/1 * * * *" \
-  --data '{"message": "Hello world!"}' \
-  --sink http://event-display.pingsource-example.svc.cluster.local
+  --schedule "*/2 * * * *" --data \
+  '{ "message": "Hello world!" }' \
+  --sink ksvc:event-display
 ```
-
 {{< /tab >}}
 {{< /tabs >}}
 
 ## (Optional) Create a PingSource with binary data
 
-Sometimes you may want to send binary data, which cannot be directly serialized in yaml, to downstream. This can be achieved by using `dataBase64` as the payload. As the name suggests, `dataBase64` should carry data that is base64 encoded.
+You might sometimes want to send binary data, which cannot be directly serialized in YAML, to
+downstream. You can do so using `dataBase64` as the payload.
+As the name suggests, `dataBase64` should carry data that is base64-encoded.
 
-Please note that `data` and `dataBase64` cannot co-exist.
+**Note:** `data` and `dataBase64` cannot co-exist.
+
+Create the event source with binary data from stdin by running:
 
 ```shell
-kubectl create -n pingsource-example -f - <<EOF
+cat <<EOF | kubectl create -f -
 apiVersion: sources.knative.dev/v1beta2
 kind: PingSource
 metadata:
   name: test-ping-source-binary
 spec:
-  schedule: "*/1 * * * *"
+  schedule: "*/2 * * * *"
   contentType: "text/plain"
   dataBase64: "ZGF0YQ=="
   sink:
     ref:
-      apiVersion: v1
+      apiVersion: serving.knative.dev/v1
       kind: Service
       name: event-display
 EOF
 ```
 
-### Verify
+## Verify the message was sent
+
+Verify that the message was sent to the Knative Eventing system by looking at message dumper logs.
+
+{{< tabs name="view-event" default="kubectl" >}}
+{{% tab name="kubectl" %}}
+
+View the logs of the event-display service by running:
```

Comment:
Be consistent with names - `event-display`

Reply:
Good catch, thanks.

Reply:
don't forget the backticks, not sure why these were removed